BusyBox for Android (#1613)
OWASP/owasp-mstg: Document/0x05b-Basic-Security_Testing.md

- APK Extractor: App to extract APKs without root.
- Frida server: Server for Frida, the dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers. See [Frida](#frida "Frida section") section below for more information.
- Drozer agent: Agent for drozer, the framework that allows you to search for security vulnerabilities in apps and devices. See [Drozer](#drozer "Drozer section") section below for more information.

> - BusyBox:  [BusyBox](https://busybox.net "BusyBox") combines multiple common Unix utilities into a small single executable. The utilities included generally have fewer options than their full-featured GNU counterparts, but are sufficient enough to provide a complete environment on a small or embedded system. BusyBox can be installed on a rooted device by downloading the BusyBox application from Google Play Store.

Reviewer: ```suggestion
- BusyBox:  BusyBox combines multiple common Unix utilities into a small single executable. The utilities included generally have fewer options than their full-featured GNU counterparts, but are sufficient enough to provide a complete environment on a small or embedded system. BusyBox can be installed on a rooted device by downloading the BusyBox application from Google Play Store. You can also download the binary directly from the [BusyBox website](https://busybox.net "BusyBox Website"). Once downloaded, make an `adb push busybox /data/local/tmp` to have the executable available on your phone. A quick overview of how to install and use BusyBox can be found in the [Busybox FAQ](https://busybox.net/FAQ.html#getting_started "Busybox FAQ").
```

> - BusyBox:  BusyBox combines multiple common UNIX utilities into a small single executable. The utilities included generally have fewer options that their full-featured GNU counterparts, but sufficient enough to provide complete environment on a small or embedded system. BusyBox can be installed on a device by downloading the binary from [BusyBox website](https://busybox.net "BusyBox Wesbite") or via Google Play Store on a rooted device.

Reviewer 2: Very nice description :) Which binary do we need for android, where should we copy it to and how to use it? Could you please add 1 or 2 sentences regarding this?

Reviewer: Couldn't find a proper installation manual when downloading the Busybox binary and there are many different suggestions on how to install Busybox binary manually. 
Let's just suggest to download the App from Play Store. It's very convenient and fulfils it's purpose.

Reviewer: @su-vikas: Sorry to change again, you can link to this here: https://busybox.net/FAQ.html#getting_started

Additional sentence:
You can also download the binary directly from the [BusyBox website](https://busybox.net "BusyBox Wesbite"). Once downloaded make an `adb push busybox /data/local/tmp` to have the executable available on your phone.  A quick overview of how to install and use busybox can be found in the [Busybox FAQ](https://busybox.net/FAQ.html#getting_started "Busybox FAQ").

> - BusyBox:  BusyBox combines multiple common UNIX utilities into a small single executable. The utilities included generally have fewer options that their full-featured GNU counterparts, but sufficient enough to provide complete environment on a small or embedded system. BusyBox can be installed on a device by downloading the binary from [BusyBox website](https://busybox.net "BusyBox Wesbite") or via Google Play Store on a rooted device.

Reviewer: ```suggestion
- BusyBox:  BusyBox combines multiple common Unix utilities into a small single executable. The utilities included generally have fewer options than their full-featured GNU counterparts, but are sufficient enough to provide a complete environment on a small or embedded system. BusyBox can be installed on a device by downloading the binary from [BusyBox website](https://busybox.net "BusyBox Website") or via Google Play Store on a rooted device.
```

> - BusyBox:  BusyBox combines multiple common Unix utilities into a small single executable. The utilities included generally have fewer options than their full-featured GNU counterparts, but are sufficient enough to provide a complete environment on a small or embedded system. BusyBox can be installed on a rooted device by downloading the BusyBox application from Google Play Store. You can also download the binary directly from the [BusyBox website](https://busybox.net "BusyBox Website"). Once downloaded, make an `adb push busybox /data/local/tmp` to have the executable available on your phone. A quick overview of how to install and use BusyBox can be found in the [Busybox FAQ](https://busybox.net/FAQ.html#getting_started "Busybox FAQ").

Reviewer 2: According to its webpage, Busybox is written "Busybox". Fixed here:

```suggestion
- Busybox:  Busybox combines multiple common Unix utilities into a small single executable. The utilities included generally have fewer options than their full-featured GNU counterparts, but are sufficient enough to provide a complete environment on a small or embedded system. Busybox can be installed on a rooted device by downloading the Busybox application from Google Play Store. You can also download the binary directly from the [Busybox website](https://busybox.net "Busybox Website"). Once downloaded, make an `adb push busybox /data/local/tmp` to have the executable available on your phone. A quick overview of how to install and use Busybox can be found in the [Busybox FAQ](https://busybox.net/FAQ.html#getting_started "Busybox FAQ").
```
